Sources/CoreFoundation/BlockRuntime/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the Swift open source project
+##
+## Copyright (c) 2024 Apple Inc. and the Swift project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of Swift project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+# Build the vendored version of the BlocksRuntime library, which is used by
+# platforms that don't support libdispatch.
+
+# Build the BlocksRuntime as an object library, shipped as a part
+# of libCoreFoundation.
+add_library(BlocksRuntime OBJECT
+    data.c
+    runtime.c)
+
+target_include_directories(BlocksRuntime PUBLIC
+    ${CMAKE_CURRENT_SOURCE_DIR}/include
+    # For CFTargetConditionals.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/../include)
+
+set_target_properties(BlocksRuntime PROPERTIES
+    POSITION_INDEPENDENT_CODE FALSE)
```

Comment:
I think you addressed the other directive in https://github.com/apple/swift-corelibs-foundation/pull/5057#discussion_r1707333900, but it looks like this one is still here - is there a particular reason why this needs to be disabled for this target, while CoreFoundation has it enabled?

Reply:
I couldn't find any reason to disable PIC. Removed it in 3b8485a75c5fd9a7f7877961ceadbc6521dcf563